### PR TITLE
sms-tool: version bump 

### PR DESCRIPTION
**Pull request by bam80**
Maintainer: me
Run tested: mipsel,  OpenWrt 23.05.0-rc3, problem in the Description solved.

Description:
Fixes:
"no SMS Messages if some of them contain From: field with special symbols":
https://github.com/4IceG/luci-app-sms-tool-js/issues/9

### CODE REVIEW

File: utils/sms-tool/Makefile [hunk 1]
```diff
@@ -6,13 +6,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=sms-tool
-PKG_RELEASE:=1
+PKG_RELEASE:=2
```
Comment (BKPepe) [inline utils/sms-tool/Makefile:9]: Why?

Comment (bam80) [reply utils/sms-tool/Makefile]: done

**Comment by bam80** [comment]
Ping, please?

**Comment by BKPepe** [comment]
Why there isn't specified version in the commit subject? I mean why did you chose to have vague commit subject? "Version bump" will be everytime, thats not helpful for history.

**Comment by bam80** [comment]
@BKPepe done

**Comment by bam80** [comment]
@BKPepe 
Thank you.

As you here, could I ask you to merge this one also:
https://github.com/openwrt/packages/pull/22144#pullrequestreview-1795466339

This was ready apart from the commit message which I fixed now.
I couldn't shout out anyone for a long time.
Thanks.

**Pull request merged**